media/css/firefox/welcome.scss

```diff
@@ -52,14 +52,19 @@
 
     .primary-cta {
         margin: 0 auto;
+
+        .mzp-c-button {
```

Comment:
Should we copy / pasta these CSS changes over to the CRO page, seeing as you're also updating that here?